src/native/components/Artistas.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, FlatList, TouchableOpacity, RefreshControl, Image, View } from 'react-native';
+import { Container, Content, Card, CardItem, Body, Text, Button } from 'native-base';
+import { Actions } from 'react-native-router-flux';
+
+import Loading from './Loading';
+import Error from './Error';
+import Header from './Header';
+import Spacer from './Spacer';
+
+const ArtistasList = ({
+  error,
+  loading,
+  artistas,
+  reFetch,
+}) => {
+  // Loading
+  if (loading) return <Loading />;
+
+  // Error
+  if (error) return <Error content={error} />;
+
+  const keyExtractor = item => item.id;
+
+  const onPress = item => Actions.artista({ match: { params: { id: String(item.id) } } });
```

Comment:
Is this `Actions.artista` code completely nothing? Nothing happens when I click.

Reply:
I think it's nothing cuz we havent done the single artist part yet. Good to keep that here till we do.